tests/test_cli.py

```diff
@@ -61,7 +61,7 @@ def test_cli_secret_write_reveal_gpg(self):
             fp.write(test_secret_content)
 
         sys.argv = ["kapitan", "secrets", "--write", "gpg:test_secret",
-                    "-f", test_secret_file,
+                    "--file", test_secret_file,
```

Comment:
We can't remove the test for the short notation as most people rely on this. Let's add an additional test that covers the long notation. If tests break, that's a side effect of the python-fire limitations or the implementation.